### PR TITLE
Release 5.4

### PR DESCRIPTION
**Pull request by jaredgalanis**

### CODE REVIEW

File: content/ember-released-5-4.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+Ember.js 5.4 introduced 1 feature:
+
+- [#20521](https://github.com/emberjs/ember.js/pull/20521) Add TypeScript support for looking up controllers in dependency injection registry
+
+#### Deprecations
+
+Ember.js 5.4 introduced 2 deprecations.
+
+- [#20529](https://github.com/emberjs/ember.js/pull/20529) - Don't rethrow string errors in handleTransitionReject
```
Comment (kategengler) [inline content/ember-released-5-4.md:43]: These are both bugfixes, not deprecations

File: content/ember-released-5-4.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+For more details on changes in Ember.js 5.4, please review the [Ember.js 5.4.0 release page](https://github.com/emberjs/ember.js/releases/tag/v5.3.0).
+
+---
+
+## EmberData
+
+EmberData is the official data persistence library for Ember.js applications.
+
+EmberData did not release a new minor version during this release cycle. Please see the [blog post](https://blog.emberjs.com/updates-to-ember-data-versioning-strategy) addressing EmberData's new versioning strategy for more information.
```
Comment (kategengler) [inline content/ember-released-5-4.md]: Might be worth summarizing here that 5.3 will become the LTS release candidate for ember-data.

Comment (jaredgalanis) [reply content/ember-released-5-4.md]: Seems like we should we mention that 5.4 will for Ember?

Comment (kategengler) [reply content/ember-released-5-4.md]: Yeah, likely. I think there's some boilerplate that goes into the LTS candidate releases iirc.

File: content/ember-released-5-4.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+Ember.js 5.4 is an incremental, backwards compatible release of Ember with bug fixes, performance improvements, and minor deprecations.
+
+#### Features
+
+Ember.js 5.4 introduced 1 feature:
+
+- [#20521](https://github.com/emberjs/ember.js/pull/20521) Add TypeScript support for looking up controllers in dependency injection registry
+
+#### Deprecations
```
Comment (kategengler) [inline content/ember-released-5-4.md]:
```suggestion
#### Bug fixes
```

File: content/ember-released-5-4.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+
+Ember.js 5.4 introduced 2 bug fixes.
+
+- [#20529](https://github.com/emberjs/ember.js/pull/20529) - Don't rethrow string errors in handleTransitionReject
+- [#20210](https://github.com/emberjs/ember.js/pull/20210) - Fix hanging tests caused by legacy rsvp code
+
+<!-- Block start: If there were no deprecations, remove this block -->
+
+Deprecations are added to Ember.js when an API will be removed at a later date. Each deprecation has an entry in the deprecation guide describing the migration path to a more stable API. Deprecated public APIs are not removed until a major release of the framework.
```
Comment (kategengler) [inline content/ember-released-5-4.md]: This section is the boilerplate for deprecations, not bug fixes.

**Review by runspired — APPROVED** [review]

**Review by kategengler — APPROVED** [review]

**Pull request merged**